src/main.cpp

```diff
@@ -441,6 +441,7 @@ void vcMain_MainLoop(vcState *pProgramState)
             if (clicked.pParent != nullptr && clicked.pItem->itemtype == vdkPNT_Media)
             {
               vdkProjectNode_SetURI(pProgramState->activeProject.pProject, clicked.pItem, pNextLoad);
+              ((vcSceneItem *)clicked.pItem->pUserData)->m_loadStatus = vcSLS_Unloaded;
```

Comment:
I don't think this is required? The OnNodeUpdate system should trigger for this change.

Reply:
Crashes without this

Reply:
That's a very concerning crash as this will crash when in collaborative mode as well.